docs-starlight/src/data/commands/stack/generate.mdx

```diff
@@ -67,3 +67,10 @@ Generates the following stack:
       - terragrunt.hcl
 
 </FileTree>
+
+**Notes**:
```

Comment:
Use [asides](https://starlight.astro.build/components/asides/) for the Starlight docs.